tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/PitSummaryView.java

```diff
@@ -16,34 +16,97 @@
 
 package org.pitest.pitclipse.ui.behaviours.pageobjects;
 
-import org.pitest.pitclipse.ui.behaviours.StepException;
-import org.pitest.pitclipse.ui.swtbot.PitNotifier;
-import org.pitest.pitclipse.ui.swtbot.PitResultsView;
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot;
+import org.eclipse.swtbot.swt.finder.waits.DefaultCondition;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotBrowser;
+import org.pitest.pitclipse.ui.view.PitView;
 
 public class PitSummaryView {
+    private SWTWorkbenchBot bot;
+    private SWTBotBrowser browser;
 
-    private PitResultsView lastResults = null;
+    public PitSummaryView(SWTWorkbenchBot bot) {
```

Comment:
Should I also review this test or shall we wait for the PR https://github.com/pitest/pitclipse/pull/167 to be merged?

Reply:
Please wait for #167.